iFixit/react-commerce
# Fix filters rendering bug

This PR closes #73 

## QA

1. Visit the Vercel PR preview
2. Visit a collection page and verify that the bug described by #73 is fixed
